style.css

```diff
@@ -0,0 +1,62 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;1,100;1,200;1,300;1,500;1,600;1,700&display=swap');
+
+html {
+  height: 718vh;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  background-color: #fff;
+}
+
+.main_container {
+  background-color: #e5e5e5;
+  max-width: 375px;
+  height: 718px;
+  background-image: url("/assets/header-shapes\ mobile.png");
```

Comment:
The background which you added, is not showing up. Here is a snapshot of your app. It may be because you have an error in the link to the file you are setting as background. 😊
![bg-not-showing](https://user-images.githubusercontent.com/33268927/170511138-66e22bc8-c508-4854-9a71-86dacbd96a17.png)

Reply:
This issue is well resolved! ✔️